src/main/scala/se/nimsa/sbx/metadata/MetaDataDAO.scala

```diff
@@ -538,9 +539,18 @@ object MetaDataDAO {
     if (whereParts.exists(_.isDefined)) " where" else ""
 
   def orderByPart(orderBy: Option[String], orderAscending: Boolean) =
-    orderBy.map(orderByValue =>
-      s""" order by "$orderByValue" ${if (orderAscending) "asc" else "desc"}""")
-      .getOrElse("")
+    orderBy.map { orderByValue =>
+
+      /*
+      MySQL is case insensitive for most collations. When joining tables for listing flat series,
+      we end up with columns `patientId` (foreign key in Series table) and patientID (DICOM
+      attribute in Patients table) and we must distinguish between these. This fix makes sure the
+      attribute is chosen over the foreign key and resolves the ambiguity.
+       */
+      val ob = if (orderByValue == "patientID") s""""Patients"."$orderByValue"""" else s""""$orderByValue""""
+
+      s""" order by $ob ${if (orderAscending) "asc" else "desc"}"""
+    }.getOrElse("")
```

Comment:
This fixes the issue with error when sorting flat series lists on patientID (#109)